sale_import_exceptions/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Sale Import Exceptions",
+    "summary": "Adds some consistency logic for sale order imports",
+    "version": "12.0.1.0.0",
+    "category": "Generic Modules/Sale",
+    "author": "Akretion",
+    "website": "https://github.com/akretion/sale-import",
+    "depends": ["sale_import_base", "sale_exception"],
+    "license": "AGPL-3",
+    "data": ["data/sale_exceptions.xml"],
+    "installable": True,
+}
```

Comment:
FYI, not mandatory for me, but in the past we rename module from plural form to singular, i.e. sale_exceptions -> sale_exception, maybe here the same ?

Reply:
Yes, thanks David